python/lsst/ts/adamSensors/model.py

```diff
@@ -34,18 +35,24 @@ def __init__(self, log=None, simulation_mode=False):
         self.range_start = -10  # zero point offset for the ADAM device
         self.simulation_mode = simulation_mode
 
-    def connect(self, ip, port):
+    async def connect(self, ip, port):
         self.clientip = ip
         self.clientport = port
         if self.simulation_mode:
-            self.client = None
+            self.client = MockModbusClient(self.clientip, self.clientport)
         else:
-            self.client = ModbusClient(self.clientip, self.clientport)
+            try:
+                self.client = ModbusClient(self.clientip, self.clientport)
+            except AttributeError:
+                raise ConnectionException(
+                    f"Unable to connect to modbus device at {self.clientip}:\
```

Comment:
Can you fix this line break the same way you did on the CSC?

Either put it all in one line or break the string into multiple concatenated strings

```
raise ConnectionException(
                    "Unable to connect to modbus device at "
                    f"{self.clientip}:{self.clientport}"
)
```